File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        int v = Integer.parseInt(st.nextToken());

        int day = (v - b) / (a - b);
        if ((v - b) % (a - b) != 0)
            day++;

        bw.write(String.valueOf(day));

        bw.flush();
        br.close();
        bw.close();
    }
}

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.cc
#include <iostream>
using namespace std;

int main() {
    long long a, b, v;
    cin >> a >> b >> v;
    long long day = 1;
    day += (v-a) / (a-b);
    if ((v-a)%(a-b) != 0) day++;
    if (a >= v) cout << 1 << "\n";
    else cout << day << "\n";
    return 0;
}